requirements.txt

```diff
@@ -4,52 +4,104 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-amqp==2.1.4               # via kombu
-anymarkup-core==0.7.0     # via anymarkup
+amqp==2.2.0               # via kombu
+anymarkup-core==0.7.1     # via anymarkup
 anymarkup==0.7.0
-beautifulsoup4==4.5.3
-BigQuery-Python==1.10.0
+beautifulsoup4==4.6.0
+bigquery-python==1.12.0
 billiard==3.5.0.2         # via celery
 boto3==1.4.4
-boto==2.45.0
-botocore==1.5.0           # via boto3, s3transfer
+boto==2.48.0
```

Comment:
Updating boto makes our downstream patch unnecessary - it fails to apply now, we should drop it.